app/scripts/modules/core/src/loadBalancer/loadBalancer.states.ts

```diff
@@ -69,7 +70,7 @@ module(LOAD_BALANCER_STATES, [
         template: '<load-balancer-filter app="$resolve.app"></load-balancer-filter>',
       },
       'master': {
-        template: '<load-balancers app="$resolve.app" style="display: flex;flex: 1 1 auto"></load-balancers>',
+        component: LoadBalancers, $type: 'react'
```

Comment:
This is how you route to a React Component.  The resolve data is bound as props (in this case, `props.app` receives the resolved `app` value).